src/Lib/ImageTransform.php

```diff
@@ -27,6 +29,28 @@ class ImageTransform
     private $Imagine;
 
     /**
+     * @var Table $table Instance of the table being used
+     */
+    protected $Table;
```

Comment:
It's passing for me with the `--standard=PSR2`. The spec doesn't mention the case of the class property, so I guess it's okay.